storage/posix/files.go

```diff
@@ -483,7 +499,7 @@ func (s *Storage) ensureVersion(version uint16) error {
 func (s *Storage) writeTreeState(size uint64, root []byte) error {
 	raw, err := json.Marshal(treeState{Size: size, Root: root})
 	if err != nil {
-		return fmt.Errorf("Marshal: %v", err)
+		return fmt.Errorf("json.Marshal: %v", err)
```

Comment:
nit: I'd only really add these package qualifiers if there's ambiguity about where in the method this error could've come from.

Reply:
Aaah, I see!